### DISCUSSION TITLE
Facing issues while connecting to snowflake using the JWT authentication.

### DISCUSSION THREAD

**Discussion by wasilmasood** [burningalchemist/sql_exporter#391] [Q&A] [answered]
The private key string which is "authenticator=SNOWFLAKE_JWT&privateKey=<your_private_key>". using the base64 URL encoded PKCS8 rsa private key string, the error in SQL exporter is "Error decoding private key using PKCS8" Need help to resolve this. @burningalchemist

**Comment by burningalchemist** [comment, marked as answer]
Hey @wasilmasood, please have a look at this thread: https://github.com/burningalchemist/sql_exporter/issues/25#issuecomment-832565000

**Reply by wasilmasood** [reply]
Thank you!!
